### ISSUE TITLE
Allow multiple Google Analytics tracking codes

### ISSUE THREAD

**Issue by krues8dr** [opengovfoundation/madison#807]
Allow users to specify a comma separated list in the `.env` file.  For instances of Madison that we're releasing as a partnership, we'll want to track our own analytics as well as allowing the partner to have their own.

**Comment by krues8dr** [comment]
https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers#working_with_multiple_trackers Is the helpful info

**Comment by krues8dr** [comment]
I've spent too much time on this already today, but can't figure out how to get a handle for `$analyticsProvider` to alter its settings before bootstrapping the app.  Clearly this requires deeper Angular knowledge than I currently possess.  Tabling this work for now.

**Comment by akuckartz** [comment]
What about Piwik ?

**Comment by krues8dr** [comment]
@akuckartz we could certainly add it, but we'd need to overhaul the config & bootstrapping a bit.  We use [angulartics](https://luisfarzati.github.io/angulartics/) which supports piwik.  Feel free to open a new issue if you need that as a feature.

**Comment by cmbirk** [comment]
Angulartics appears to be able to support multiple GA accounts out of the box: 

https://github.com/angulartics/angulartics/blob/8371cc04925147204c74ad03ad8c984bd8408819/CHANGELOG.md#features

I just have to figure out how that operates

**Comment by krues8dr** [comment]
@cmbirk I already did all of the work here, it's just a question of adding that one line I've got commented out and can't figure out how to get access to the provider to add the property:

https://github.com/opengovfoundation/madison/commit/7332eb1b27b78004fddbc5d27cb70d8e66d2edcd#diff-f5260081ec80c28a01bd5809279dbfa4R153

**Comment by cmbirk** [comment]
So I think the actual _angular way_ to do this is attach a vendor service to the app that holds this information.  That would be accessible the same way we get the annotation service from the annotator plugin.

**Comment by cmbirk** [comment]
No, I don't think that's true.  I think the model is to go the provider / service route like I did in the prompts module.
